arpack.pc.in

```diff
@@ -7,6 +7,5 @@ Name: @PACKAGE_NAME@
 Description: Collection of Fortran77 subroutines designed to solve large scale eigenvalue problems
 Version: @PACKAGE_VERSION@
 URL: @PACKAGE_URL@
-Libs: -L${libdir} -larpack@LIBSUFFIX@
-Libs.private: @LAPACK_LIBS@ @BLAS_LIBS@
-Cflags: -I${includedir}/arpack
+Libs: -L${libdir} -larpack@LIBSUFFIX@ @LAPACK_LIBS@ @BLAS_LIBS@
```

Comment:
This is wrong; LAPACK and BLAS are private. Moving them to public causes overlinking of libraries that use arpack. If you are using a static build of arpack, you need to use `pkg-config --static` to grab the private flags.

Reply:
what do you mean by private ?

Reply:
Private meaning the `Libs.private` field that they used to be in.

Reply:
Couldn't link without it at the time. Adding non reg test for that would be good.

Reply:
ok, please provide patches :)